csrc/transformer/inference/csrc/gelu.cu

```diff
@@ -212,7 +212,7 @@ __global__ void fused_bias_residual(__half* input,
                                     __half* attn_bias,
                                     int total_count,
                                     int intermediate_size,
-                                    int mp_size,
+                                    float mp_size,
```

Comment:
I am confused why mp_size should be a float rather than int. Perhaps add some comment explaining this?

Reply:
That's true, the MP_Size should be integer. But this one is just 1/MP_Size and that's why it needs to be float, so better name would be mp_scale.